Index: resources/views/testtesttest2.blade.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/views/testtesttest2.blade.php	(revision c65d7253efff3d6e52320348bc4eb67718762d1d)
+++ resources/views/testtesttest2.blade.php	(date 1590998826550)
@@ -6,7 +6,7 @@
     <link href="{{asset('/css/bootstrap.css')}}" rel="stylesheet" type="text/css">
     <link href="{{asset('/css/main.css')}}" rel="stylesheet" type="text/css">
 </head>
-
+выстрел шариком под углом по нажатию пробела
 <style type="text/css">canvas {
         border: 1px solid black;
         float: left;
Index: resources/views/testtesttest.blade.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/views/testtesttest.blade.php	(revision c65d7253efff3d6e52320348bc4eb67718762d1d)
+++ resources/views/testtesttest.blade.php	(date 1590998520894)
@@ -6,7 +6,7 @@
     <link href="{{asset('/css/bootstrap.css')}}" rel="stylesheet" type="text/css">
     <link href="{{asset('/css/main.css')}}" rel="stylesheet" type="text/css">
 </head>
-
+Шарики-планеты-атомы, черная дыра
 <style type="text/css">canvas {
         border: 1px solid black;
         float: left;
